week-1/E-strings/exercise.js

```diff
@@ -1,3 +1,11 @@
 // Start by creating a variable `message`
+var message = "This is a string";
+var messageType = typeof message;
+console.log(message);
+console.log(typeof message);
 
+var message = 7;
```

Comment:
Do you remember in which cases we use the keyword `var` and in which cases we just leave it out?